nevergrad/functions/images/test_core.py

```diff
@@ -7,15 +7,56 @@
 import numpy as np
 import nevergrad as ng
 from . import core
+from torchvision.models import resnet50
+import torch.nn as nn
+import torch
+import torchvision.models
+from torchvision.models.resnet import model_urls
+
+model_urls['resnet50'] = model_urls['resnet50'].replace('https://', 'http://')
+
+class Normalize(nn.Module):
```

Comment:
this can (should?) be removed from here

Reply:
More precisely, this and almost everything in this file should be in image/core.py and called in __init__. The test should not specify the classifier or whatever; the classifier should be created from __init__ in image/core.py.